kolibri/plugins/coach/assets/src/views/home/HomePage/ActivityBlock.vue

```diff
@@ -5,11 +5,15 @@
     :allLinkRoute="$router.getRoute('HomeActivityPage')"
     :showAllLink="notifications.length > 0"
   >
-    <template v-slot:title>
-      {{ $tr('classActivityLabel') }}
-    </template>
+
+    <KLabeledIcon slot="title" :label="$tr('classActivityLabel')" />
```

Comment:
We should keep the original `v-slot` syntax, since it will be Vue 3 compatible. Even better is the `#` short hand
```suggestion
    <KLabeledIcon #title :label="$tr('classActivityLabel')" />
```